Airports/EGCC/Sector_Ownership.txt

```diff
@@ -46,7 +90,15 @@ ALTOWNER:Observing MAN_SE_CTR:CCTS:CCT:CCF:CCS:CCN:O:LMEN:PCSE:PCE:PC:LNW:LN:L
 ALTOWNER:Observing London FIR:O:LMEN:CCTS:CCT:CCF:CCS:CCN:PCSE:PCE:PC:LNW:LN:L
 ALTOWNER:Mentoring TWR/APP:CCTS:CCTM:CCT:CCF:CCSM:CCS:CCN:PCSE:PCE:PC:LNW:LN:L
 ALTOWNER:Mentoring FIN:CCTM:CCTS:CCT:CCF:CCSM:CCS:CCN:PCSE:PCE:PC:LNW:LN:L
-BORDER:CCTWR
+BORDER:CCTWR1E_CCTWR2E:CCTWR2E_OUTER
+DEPAPT:EGCC
+ARRAPT:EGCC
+ACTIVE:EGCC:05L
+ACTIVE:EGCC:05R
+
+SECTOR:CCATZ:0:2000
+OWNER:CCTS:CCT:CCF:CCS:CCN:PCSE:PCE:PC:LNW:LN:L
+BORDER:CCATZ
 DEPAPT:EGCC
 ARRAPT:EGCC
```

Comment:
```suggestion
```
Why are you defining this as a sector? I think it might break sector inbound/outbound predictions, and seems unnecessary if you define the ATZ's display parameters as suggested in the boundaries file.